Resources/Prototypes/_DV/Entities/Objects/Misc/land_mine.yml

```diff
@@ -0,0 +1,27 @@
+- type: entity
+  name: explosive mine
+  suffix: DeltaV
+  parent: BaseLandMine
+  id: LandMineHE
+  components:
+  - type: ExplodeOnTrigger
+  - type: Explosive
+    explosionType: Default
+    maxIntensity: 21 # DeltaV - 10 > 20
+    intensitySlope: 10 # DeltaV - 3 > 10
+    totalIntensity: 120 # about a ~4 tile radius
+    canCreateVacuum: false
+
+- type: entity
+  name: shrapnel mine
+  suffix: DeltaV
```

Comment:
there is no upstream shrapnel mine so suffix isnt needed
same field ordering applies